app/app.go

```diff
@@ -315,6 +330,7 @@ func NewWbServiceApp(logger log.Logger, db dbm.DB, config *config.VMConfig, base
 		multisig.ModuleName,
 		vm.ModuleName,
 		genutil.ModuleName,
+		oracle.ModuleName,
```

Comment:
```
// Sets the order of Genesis - Order matters, genutil is to always come last
// NOTE: The genutils moodule must occur after staking so that pools are
```
Fix please :)